packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/FishjamClientInternal.kt

```diff
@@ -142,32 +154,36 @@ internal class FishjamClientInternal(
           t: Throwable,
           response: Response?
         ) {
-          listener.onSocketError(t, response)
-          commandsQueue.clear("Socket error")
+          listener.onSocketError(t)
+          commandsQueue.onDisconnected()
         }
       }
 
-    commandsQueue.addCommand(
-      Command(CommandName.CONNECT, ClientState.CONNECTED) {
-        val request = Request.Builder().url(config.websocketUrl).build()
-        val webSocket =
-          OkHttpClient().newWebSocket(
-            request,
-            websocketListener
-          )
-
-        this.webSocket = webSocket
-      }
-    )
+    coroutineScope.launch {
+      commandsQueue.executeCommand(
```

Comment:
I find naming here a little... confusing? Should this method be still called `addCommand` as it is not guarantee to always execute passed command?
I guess something like `addAndMaybeExecuteCommand` would be more correct - but it looks strange :p